repos/MSBuildLocator.proj

```diff
@@ -16,6 +16,7 @@
       <BuildCommandArgs>$(BuildCommandArgs) /v:$(LogVerbosity)</BuildCommandArgs>
       <BuildCommandArgs>$(BuildCommandArgs) $(RedirectRepoOutputToLog)</BuildCommandArgs>
       <BuildCommandArgs>$(BuildCommandArgs) /p:MicrosoftSourceLinkVersion=$(MicrosoftSourceLinkVersion)</BuildCommandArgs>
+      <BuildCommandArgs>$(BuildCommandArgs) /p:PublicSign=$(PublicSign)</BuildCommandArgs>
```

Comment:
Should I also consider passing along other related arcade properties like `SignAssemblies`?

Reply:
@crummel - Can you share your thoughts on this one?  I tend to think no.

Reply:
ping @crummel

Reply:
No, I think this should be fine without the extra properties.

Reply:
Actually taking a second look at this they are pretty tied together so they probably should all be passed, sorry.

Reply:
Looking at the properties that are defined at https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Arcade.Sdk/tools/StrongName.targets:

```
    Writes variables:
      DelaySign
      PublicSign
      PublicKey
      PublicKeyToken
      AssemblyOriginatorKeyFile
```

I can see passing `DelaySign`, `PublicSign` properties. The other three seem like they would override whatever the public key, token and the snk file that the repo is using. That doesn't seem like a good idea to me. Thoughts?

Reply:
Okay, looked at other projects and it seems like they are only passing hose 2 properties too (if not just one).

Reply:
Yes, I think that makes sense.